junit-runtime/src/main/scala/org/scalajs/junit/JUnitTestBootstrapper.scala

```diff
@@ -0,0 +1,78 @@
+package org.scalajs.junit
+
+import java.lang.annotation.Annotation
+
+import org.junit.FixMethodOrder
+
+import scala.scalajs.js.annotation.JSExportDescendentObjects
+
+@JSExportDescendentObjects
+sealed trait JUnitTestBootstrapper {
```

Comment:
This must not be sealed, as the compiler plugin generates subclasses of that trait.